src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -14,15 +14,19 @@
     /** Help information should be shown to the user. */
     private final boolean showHelp;
 
+    /** Statistics should be shown to the user. */
+    private final boolean showStats;
```

Comment:
Naming of boolean variables sounds a bit unintuitive, though I'm not sure how they can be improved either.